spec.html

```diff
@@ -7,6 +7,60 @@
 contributors: Chengzhong Wu, Justin Ridgewell
 </pre>
 
+<emu-clause id="sec-ecmascript-data-types-and-values">
+  <h1>ECMAScript Data Types and Values</h1>
+
+  <emu-clause id="sec-ecmascript-specification-types">
+    <h1>ECMAScript Specification Types</h1>
+
+    <ins class="block">
+    <emu-clause id="sec-asynccontext-mapping-record-specification-type">
+      <h1>The Async Context Mapping Record Specification Type</h1>
+      <p>The <dfn variants="Async Context Mapping Records">Async Context Mapping Record</dfn> type is used to represent an AsyncLocal value mapping in the surrounding Agent's [[AsyncContextMapping]].</p>
+      <p>A Async Context Mapping Record's fields are defined by <emu-xref href="#table-asynccontext-mapping-record-fields"></emu-xref>.</p>
+
+      <emu-table id="table-asynccontext-mapping-record-fields" caption="Async Context Mapping Record Fields">
+        <table>
+          <tr>
+            <th>
+              Field Name
+            </th>
+            <th>
+              Value
+            </th>
+            <th>
+              Meaning
+            </th>
+          </tr>
+          <tr>
+            <td>
+              [[AsyncContextKey]]
+            </td>
+            <td>
+              an AsyncLocal instance
+            </td>
+            <td>
+              The name of the AsyncLocal instance.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              [[AsyncContextValue]]
+            </td>
+            <td>
+              an ECMAScript language value
+            </td>
+            <td>
+              The default value of the AsyncLocal instance when no entry is found in the mapping.
```

Comment:
The descriptions for both fields in this table are wrong, the current text is copied from the `AsyncLocal` internal slots table.